addons/crm/views/crm_lead_views.xml

```diff
@@ -9,8 +9,8 @@
                         <button name="action_set_won_rainbowman" string="Won"
                             type="object" class="oe_highlight" data-hotkey="w" title="Mark as won"
                             attrs="{'invisible': ['|','|', ('active','=',False), ('probability', '=', 100), ('type', '=', 'lead')]}"/>
-                        <button name="%(crm.crm_lead_lost_action)d" string="Lost" data-hotkey="l" title="Mark as lost"
```

Comment:
Note that in 'crm_lead_lost_action' we could add in context: default_lead_ids: [active_id], if we want to be able to call this action directly from a form view.

Reply:
Well, why not ... would not remove it at upgrade (people might use it in some automated flow), but I don't think it is necessary anymore since the action itself does the same thing, indeed.